### DISCUSSION TITLE
June 29, 2022

### DISCUSSION THREAD

**Discussion by lastzero** [photoprism/photoprism#2479] [General]
With this update, the user interface offers a [faster and smoother scrolling experience](https://github.com/photoprism/photoprism/pull/2433), as well as [direct streaming](https://github.com/photoprism/photoprism/issues/2461) of OGV, VP8, VP9, AV1, WebM and HEVC videos if they do not exceed the [configured bitrate limit](https://docs.photoprism.app/getting-started/config-options/#file-converters). Special thanks to [Heiko Mathes](https://github.com/heikomat) and [Andre Carrera](https://github.com/acarrera94) for their contributions!

#### What is new?

- UX: [Faster and smoother scrolling experience in albums and search results](https://github.com/photoprism/photoprism/pull/2433)
- Videos: [Direct streaming of OGV, VP8, VP9, AV1, WebM, and HEVC where supported](https://github.com/photoprism/photoprism/issues/2461)
- Videos: [Fixed incorrect frame rate when using NVIDIA hardware transcoding](https://github.com/photoprism/photoprism/issues/2442)
- Sharing: [Fixed the spacing of the top navigation toolbar on small screens](https://github.com/photoprism/photoprism/pull/2430)
- RAW: [Display actual dimensions as Exif metadata can be wrong, e.g. for `.NEF` files](https://github.com/photoprism/photoprism/issues/2447)
- WebDAV: [Endpoints have been disabled in public mode as they cannot be used](https://github.com/photoprism/photoprism/issues/2464)
- API: [Maximum number of search results has been increased to 100,000 files](https://github.com/photoprism/photoprism/commit/b6d32f828b6f4beea504e433f67c5395df178053)
- CLI: [Config command also lists `disable-webdav` and `http-compression`](https://github.com/photoprism/photoprism/issues/2476)
- Documentation: [Added notes about manual session invalidation and other known issues](https://docs.photoprism.app/known-issues/)
- Translations: [Updated Arabic](https://github.com/photoprism/photoprism/pull/2458/files), [Dutch, Polish](https://github.com/photoprism/photoprism/pull/2435/files), [Japanese, Chinese](https://github.com/photoprism/photoprism/pull/2445/files), [French](https://github.com/photoprism/photoprism/commit/0f0d2b4df05ff0534c9435a7802e47672f0adcb7), [German, and Italian](https://github.com/photoprism/photoprism/commit/49b9c4afb76ea316620194142e836003b0298f23)

<hr /><em>This discussion was created from the release <a href='https://github.com/photoprism/photoprism/releases/tag/220629-5d7448d2'>June 29, 2022</a>.</em>

**Comment by agramner** [comment]
The scrolling is way smoother now, great improvement of a key feature. Nice work everyone involved! :1st_place_medal:

**Comment by bm55** [comment]
Testing videos from iPhone shows ".mov cannot be streamed directly, average bitrate 0.0 MBit/s". I added following in my docker composer:

PHOTOPRISM_FFMPEG_BITRATE: 80

My goal is to play the video without transcoding over local network. Any suggestion?


time="2022-06-29T22:40:08Z" level=debug msg="http: GET /api/v1/t/b509d57d885b407960695f0ced027919e5edcdce/20f59a63/fit_720 (200) [477.154µs]"
time="2022-06-29T22:40:09Z" level=debug msg="video: 2022/06/20220629_223324_1A273790.mov cannot be streamed directly, average bitrate 0.0 MBit/s"
time="2022-06-29T22:40:09Z" level=debug msg="media: exiftool json files disabled"
time="2022-06-29T22:40:09Z" level=debug msg="metadata: exif not supported in 20220629_223324_1A273790.mov"
time="2022-06-29T22:40:09Z" level=debug msg="media: exif not supported"
time="2022-06-29T22:40:09Z" level=debug msg="media: exif not supported"
time="2022-06-29T22:40:09Z" level=debug msg="media: exif not supported"
time="2022-06-29T22:40:09Z" level=debug msg="media: exif not supported"
time="2022-06-29T22:40:09Z" level=info msg="libx264: transcoding 2022/06/20220629_223324_1A273790.mov to avc"
time="2022-06-29T22:40:39Z" level=info msg="libx264: created 20220629_223324_1A273790.mov.avc [30.238737351s]"

**Reply by lastzero** [reply]
Looks metadata like duration and codec are missing. Do you have Exiftool disabled? Could you try what information Exiftool shows if you use it directly?

**Reply by lastzero** [reply]
Yep, logs even show that directly:
```
media: exiftool json files disabled
```
Please enable Exiftool for proper video support as mentioned in our docs.

**Reply by bm55** [reply]
Thanks for pointing to me about the exiftool. I think I disabled that to keep sidecar size smaller when I initially created the docker

**Reply by lastzero** [reply]
JSON files are tiny compared to the videos you want to watch :)

**Reply by bm55** [reply]
Is there an option to disable the transcoding when someone tries to play the video on non supported device/browser?

**Reply by lastzero** [reply]
Rather not... I mean, you could disable FFmpeg, but then you wouldn't have thumbnails either. Or you could transcode all videos to AVC in advance and delete the originals for maximum compatibility and minimum storage requirements. Or you could remove the transcoding functionality and compile your own version that does that.
